Voice Assistant.js

```diff
@@ -0,0 +1,107 @@
+let personName = " ";
+let toDoList = [];
+
+function getReply(command) {
+  if (command.startsWith("Hello my name is")) {
```

Comment:
If `command` would be `hello my name is Valentin`, would this test be `true` or `false`?

Reply:
It will be true

Reply:
Is it?
<img width="242" alt="image" src="https://github.com/oyelajakenny/HYF-Homework/assets/5467758/9884caa5-0066-4496-b1c0-db3a7e9e176e">